assets/scss/04-templates/_information-details.scss

```diff
@@ -1,13 +1,3 @@
-.ma__information-details {
-
-  &__content {
-
-    & > * {
-      @include ma-component-spacing;
-    }
-  }
-}
-//theme
```

Comment:
Theme specific styles are separated from the common styles, so everybody knows where to touch for theme related changes.

Reply:
The same suggestion as the `assets/scss/03-organisms/_stacked-row-section.scss`'s.

- Remove L. 9 `}`.
- Keep L. 10`//theme`.
- Remove LL. 11-12.
   ```
   .ma__information-details {

   ```